test/CMakeLists.txt

```diff
@@ -29,6 +38,10 @@ if(NOT TARGET gtest)
     gtest_hide_internal_symbols)
 endif()
 
+if(NOT TARGET mmio)
```

Comment:
if(EXISTS mmio) is probably what you are looking for. mmio is not a target at this point is it?

Reply:
It is actually, it is a static library.